service/src/main/java/com/epam/esm/validator/GiftCertificateValidator.java

```diff
@@ -0,0 +1,95 @@
+package com.epam.esm.validator;
+
+import com.epam.esm.entity.GiftCertificate;
+
+import java.util.Objects;
+
+/**
+ * The type GiftCertificateValidator.
+ *
+ * @author Verbovskiy Sergei
+ * @version 1.0
+ */
+public class GiftCertificateValidator {
+    private static final String NAME_PATTERN = "^[0-9a-zA-Z-_]{1,45}$";
+    private static final String DESCRIPTION_PATTERN = "[^*<>\\\\/{|}]{0,100}";
```

Comment:
Please use javax.validation library for that

Reply:
> Please use javax.validation library for that

Thank you, fixed